src/test/java/redis/clients/jedis/ClusterPipeliningTest.java

```diff
@@ -55,8 +53,8 @@ public static void setUp() throws InterruptedException {
     node3.flushAll();
 
     // add nodes to cluster
-    node1.clusterMeet(LOCAL_IP, nodeInfo2.getPort());
-    node1.clusterMeet(LOCAL_IP, nodeInfo3.getPort());
+    node1.clusterMeet(JedisClusterTestUtil.getClusterIp(2), 6379);
```

Comment:
Could we possibly introduce the dotenv pattern here in general? Maybe this is overthinking it, but we did this [here](https://github.com/RedisLabs/module-tests), using an overloader [this way](https://github.com/RedisLabs/module-tests/blob/main/helpers/config.py)

Reply:
I'll have a look. We should decouple the running Redis instances from the test code, via some configuration.